src/System.Drawing.Common/tests/System.Drawing.Common.Tests.csproj

```diff
@@ -43,6 +52,12 @@
     <Content Include="bitmaps\96x96_one_entry_8bit.ico">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
+    <Content Include="bitmaps\milkmateya01.emf">
```

Comment:
nope, i forgot remove it. They are used in Metafile Tests which i didn't attach because i'm still working on.